pagopa/pm/spec.json

```diff
@@ -1818,6 +1818,55 @@
           }
         ]
       }
+    },
+    "/v2/wallet/{idWallet}/payment-status": {
+      "put": {
+        "summary": "change the ability to pay",
+        "operationId": "changePayOption",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "idWallet",
+            "type": "integer",
+            "required": true,
+            "description": "Numeric ID of the wallet"
+          },
+          {
+            "in": "body",
+            "name": "payRequest",
+            "description": "New Status required",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WalletPaymentstatus"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "New wallet State",
+            "schema": {
+              "$ref": "#/definitions/WalletV2Response"
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "content": {}
```

Comment:
```suggestion
```
In order to remove the error when imported in swagger editor (not used the content)